vexination/index/src/lib.rs

```diff
@@ -1021,6 +1032,20 @@ mod tests {
         });
     }
 
+    #[tokio::test]
+    async fn test_ngrams() {
+        assert_search(|index| {
+            let result = search(&index, "title:openssl");
+            assert_eq!(result.0.len(), 2);
+
+            let result = search(&index, "title:open");
+            assert_eq!(result.0.len(), 2);
+
+            let result = search(&index, "title:ssl");
```

Comment:
Would searching for just "ssl" work, too, since title is among the default? Or will the prefix always be required for the partial word search?

Reply:
Yes, it would. I'm just using title here because it makes it easier to look at the test data and understand why/why not the search matches.